hrui/tests/phpunit/bootstrap.php

```diff
@@ -4,6 +4,14 @@
 ini_set('safe_mode', 0);
 eval(cv('php:boot --level=classloader', 'phpcode'));
 
+// Add other extensions to include path to allow use of CRM/* classes
+$civiHRRoot = realpath(__DIR__ . '/../../../');
+$extensionDirs = glob($civiHRRoot . '/*', GLOB_ONLYDIR);
+foreach ($extensionDirs as $extensionDir) {
+  $include_path = $extensionDir . PATH_SEPARATOR . get_include_path( );
```

Comment:
I don't think we need to include all the extension directories If we find a better way to solve the dependency on relationship types.